custom_components/ocpp/const.py

```diff
@@ -39,7 +39,7 @@
 DEFAULT_MAX_CURRENT = 32
 DEFAULT_PORT = 9000
 DEFAULT_SKIP_SCHEMA_VALIDATION = False
-DEFAULT_FORCE_SMART_CHARGING = False
+DEFAULT_FORCE_SMART_CHARGING = True
```

Comment:
The default should not be changed in the code base. Simply remove ocpp integration from HA, reinstall and select the force smart charging option.

Reply:
This change wasn't meant o be part of the pull request was just so users of the current ProjectEV version didn't have to change this setting after pulling your updates into the code.